server/src/controllers/authController.ts

```diff
@@ -120,22 +120,85 @@ export const idTokenHandler = async (req: Request, res: Response) => {
     maxAge: 3600000
   });
 
-  const userCookie: UserCookie = {
-    id: userId,
-    google_id: userGoogleId,
-    firstname: userFirstname,
-    lastname: userLastname,
-    email: userEmail,
-    picture: userPicture
-  };
-  logger.info({ message: "idTokenHandler userCookie", value: userCookie });
+  res.status(200).send();
```

Comment:
What you expected to send back?

Reply:
We don't need to send anything back.
The HTTP-Only Cookie (with Custom JWT inside) is included in the Response Header
When the Browser recieves the Response from the backend it automatically stores the Cookie.

Reply:
These are the options we could use:

`res.send();`
`res.status(200).end(); `
`res.sendStatus(200);`

`res.send():`

    Sends empty response body
    Sets status code to 200
    Does NOT allow further modification of response (cannot call res.json() etc after)

`res.status(200).end():`

    Does NOT send response body
    Manually sets status code to 200
    Does NOT allow further modification of response

`res.sendStatus(200):`

    Does NOT send response body
    Sets status code to 200
    Does NOT allow further modification of response

Reply:
```suggestion
  res.sendStatus(200);
```